### ISSUE TITLE
Handling of unexpected interruptions during upload

### ISSUE THREAD

**Issue by pcaversaccio** [daita-technologies/daita-python-library#6]
We should take care of proper error handling after unexpected interruptions (e.g. after hitting ctrl + c) during upload. 

The goal should be that everything is uploaded until the breaking point.

**Comment by BEdaita** [comment]
Hi Pascal, currently I am reviewing this issue. Because the process when using the library often has two process
Firstly, The library will send information data to the server (create presign URL , validate extension) after that the library gets push data to s3 . If we handle unexpected interruption in step two, I will change workflow and I need to time to investigate.

**Comment by pcaversaccio** [comment]
@BEdaita I don't think we need to change the workflow right now. What we need to do is to inform the user in the terminal if he/she hits `ctrl + c` the data will be still pushed to storage if the process is in step two.

**Comment by BEdaita** [comment]
@pcaversaccio Do you mean we make a notification for the client and continue the process ?

**Comment by pcaversaccio** [comment]
yes, that's the lowest hanging fruit right now without overengineering everything - but we keep this issue open for later to handle everything properly.

**Comment by BEdaita** [comment]
Ok, thank you so much. I cleared the issue.
